lib/facter/docker.rb

```diff
@@ -80,9 +80,20 @@ def interfaces
 Facter.add(:docker_worker_join_token) do
   setcode do
     if Facter::Core::Execution.which('docker')
```

Comment:
I think we could make the facts more readable if we confine them on the docker binary, instead of this if:
```
confine { Facter::Core::Execution.which('docker') }
```

Reply:
Done